R/pkg/R/mllib.R

```diff
@@ -870,7 +872,7 @@ setMethod("summary", signature(object = "LogisticRegressionModel"),
 #' @param ... additional arguments passed to the method.
 #' @return \code{spark.mlp} returns a fitted Multilayer Perceptron Classification Model.
 #' @rdname spark.mlp
-#' @aliases spark.mlp,SparkDataFrame-method
+#' @aliases spark.mlp,SparkDataFrame-method,formula-method
```

Comment:
this should be `#' @aliases spark.mlp,SparkDataFrame,formula-method` with one `-method` at the end